src/components/MoneyRequestHeader.tsx

```diff
@@ -116,6 +126,21 @@ function MoneyRequestHeader({
         }
     };
 
+    const getPendingType = () => {
+        if (TransactionUtils.isExpensifyCardTransaction(transaction) && TransactionUtils.isPending(transaction)) {
+            return {pendingType: 'PENDING', pendingTitle: translate('iou.pending'), pendingDescription: translate('iou.transactionPendingText')};
+        }
+        if (isScanning) {
+            return {pendingType: 'SCANNING', pendingTitle: ReceiptScan, pendingDescription: translate('iou.receiptScanInProgressDescription')};
+        }
+        if (TransactionUtils.hasPendingRTERViolation(TransactionUtils.getTransactionViolations(transaction?.transactionID ?? '', transactionViolations))) {
+            return {pendingType: 'RTER', pendingTitle: Expensicons.Hourglass, pendingDescription: translate('iou.pendingMatchWithCreditCardDescription')};
+        }
+        return {};
+    };
```

Comment:
To consider: adding `useCallback`.

Reply:
- it's based on the transaction, which is one of the key elements that change in this components props, so probably there wouldn't be any major benefits coming from this change
- probably useMemo would be more suitable, but as I stated, there aren't any major benefits and since there aren't any issues either we should avoid it [as stated in the docs](https://react.dev/reference/react/useMemo#should-you-add-usememo-everywhere)